backend/internal/service/webauthn_service.go

```diff
@@ -120,17 +120,25 @@ func (s *WebAuthnService) VerifyRegistration(ctx context.Context, sessionID, use
 		tx.Rollback()
 	}()
 
-	// Load & delete the session row
+	// Load the session row first (MySQL doesn't support RETURNING in DELETE)
 	var storedSession model.WebauthnSession
 	err := tx.
 		WithContext(ctx).
-		Clauses(clause.Returning{}).
-		Delete(&storedSession, "id = ?", sessionID).
+		First(&storedSession, "id = ?", sessionID).
```

Comment:
Do we need to READ LOCK the row?

Reply:
Not on SQLite (a transaction blocks the entire database).

Good q on whether this is needed on PG/My

Reply:
The goal is to ensure the session is only used once, right?